arch/x86/kvm/vmx/pkvm/hyp/ept.c

```diff
@@ -489,8 +489,16 @@ static struct pkvm_mm_ops shadow_ept_mm_ops = {
  * protected VM when there are passthrough devices, and in this
  * case the memory is pinned, and the mapping is not allowed to
  * be removed from pgstate_pgt.
+ *
+ * Also pgstate_pgt_mm_ops is reused for shadow IOMMU page tables
```

Comment:
As `pgstate_pgt_mm_ops` becomes used not just for pgstate_pgt but also for iommu shadow pgt, maybe we should rename it to avoid confusion? But I'm not sure what name to use to describe both pgstate_pgt and iommu_spgt and at the same time distinguish them both from shadow_ept. Perhaps just `shadow_ept_mm_ops_notlbflush` (and correspondingly `shadow_ept_mm_ops_notlbflush_noncoherency`)?

Reply:
Ok, I think I like this idea. It reflects the fact that both tables have EPT format.

Reply:
Renamed to `shadow_sl_iommu_pgt_mm_ops`.